packages/grid/grid.0.1.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "Jean-Christophe.Filliatre@cnrs.fr"
+authors: "Jean-Christophe Filliâtre"
+synopsis: "A tiny library for two-dimensional arrays"
+description: "Elementary functions over two-dimensional arrays. Not a linear algebra library."
+license: "LGPL-2.1"
```

Comment:
Do you want `LGPL-2.1-only` or `LGPL-2.1-or-later` ?
ref: https://spdx.org/licenses/

Reply:
fixed (to LGPL-2.1-or-later)